setup/bindir/cloud-setup-databases.in

```diff
@@ -188,7 +188,7 @@ for full help
         sys.exit(1)
 
     def setupDBSchema(self):
-        if not self.rootuser:
+        if not self.options.createschema and not self.rootuser:
```

Comment:
options.createschemaonly makes it more readable, can you please change that.

Reply:
ok